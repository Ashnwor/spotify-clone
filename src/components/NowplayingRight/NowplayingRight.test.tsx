import React from 'react';
import { shallow } from 'enzyme';
import NowPlayingRight from './NowplayingRight';

it('runs correctly', () => {
  const wrapper = shallow(<NowPlayingRight />);
  expect(wrapper).toMatchSnapshot();
});
