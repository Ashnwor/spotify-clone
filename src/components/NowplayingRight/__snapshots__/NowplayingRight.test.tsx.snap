// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="nowplaying-right-container"
>
  <div
    className="icon-container"
  >
    <SvgPlaylist
      className="icon"
    />
  </div>
  <div
    className="icon-container"
  >
    <SvgDevices
      className="icon"
    />
  </div>
  <div
    className="icon-container"
  >
    <SvgVolume
      className="icon"
    />
  </div>
  <div
    className="slider-container"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <Slider
      axis="x"
      disabled={false}
      onChange={[Function]}
      onDragEnd={[Function]}
      onDragStart={[Function]}
      styles={
        Object {
          "active": Object {
            "backgroundColor": "#b3b3b3",
          },
          "thumb": Object {
            "height": 12,
            "visibility": "hidden",
            "width": 12,
          },
          "track": Object {
            "backgroundColor": "#535353",
            "borderRadius": 2,
            "height": 4,
            "width": 84,
          },
        }
      }
      x={50}
      xmax={1}
      xmin={0}
      xreverse={false}
      xstep={0.1}
      y={50}
      ymax={100}
      ymin={0}
      yreverse={false}
      ystep={1}
    />
  </div>
</div>
`;
